tests/fixtures/certificates/server.crt

```diff
@@ -1,22 +1,110 @@
 -----BEGIN CERTIFICATE-----
```

Comment:
Here I'm storing the entire shotgunlocalhost.com cert chain so we can test the server with that.